src/Tests/Microsoft.DotNet.Tools.Tests.Utilities.Tests/NuGetTransientErrorDetectorTests.cs

```diff
@@ -0,0 +1,141 @@
+      C:\h\w\B79F0A31\p\d\sdk\6.0.100-ci\NuGet.targets(131,5): error : The feed 'dotnet-public [https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json]' lists package 'runtime.win-x64.Microsoft.NETCore.DotNetHostResolver.2.1.24' but multiple attempts to download the nupkg have failed. The feed is either invalid or required packages were removed while the current operation was in progress. Verify the package exists on the feed and try again. [C:\h\w\B79F0A31\t\dotnetSdkTests\zvrozon1.u4m\RunFromOutput---802469DB\RunFromOutputFolderWithRID_netcoreapp2.1\RunFromOutputFolderWithRID_netcoreapp2.1.csproj]
+      C:\h\w\B79F0A31\p\d\sdk\6.0.100-ci\NuGet.targets(131,5): error :   Unable to find package 'runtime.win-x64.Microsoft.NETCore.DotNetHostResolver.2.1.24'. [C:\h\w\B79F0A31\t\dotnetSdkTests\zvrozon1.u4m\RunFromOutput---802469DB\RunFromOutputFolderWithRID_netcoreapp2.1\RunFromOutputFolderWithRID_netcoreapp2.1.csproj]";
+            NuGetTransientErrorDetector.IsTransientError(input).Should().BeTrue();
+        }
+
+        [Fact]
+        public void NoTransientError()
+        {
+            string input =
+                @"  Writing /var/folders/8_/2zhlfd4j3nn3pgm6vg64h5580000gn/T/tmpLxIOGq.tmp
+info : Adding PackageReference for package 'Newtonsoft.Json' into project '/Users/williamlee/Documents/temp/console/console.csproj'.
+info : Restoring packages for /Users/williamlee/Documents/temp/console/console.csproj...
+info : Package 'Newtonsoft.Json' is compatible with all the specified frameworks in project '/Users/williamlee/Documents/temp/console/console.csproj'.
+info : Committing restore...
+info : Generating MSBuild file /Users/williamlee/Documents/temp/console/obj/console.csproj.nuget.g.props.
+info : Generating MSBuild file /Users/williamlee/Documents/temp/console/obj/console.csproj.nuget.g.targets.
+info : Writing assets file to disk. Path: /Users/williamlee/Documents/temp/console/obj/project.assets.json
+log  : Restored /Users/williamlee/Documents/temp/console/console.csproj (in 120 ms).
+
+Build started 3/10/2021 1:53:51 PM.
+     1>Project  on node 1 (Restore target(s)).
+     1>_GetAllRestoreProjectPathItems:
+         Determining projects to restore...
+       Restore:
+         Restoring packages for /Users/williamlee/Documents/temp/console/console.csproj...
+         Committing restore...
+         Generating MSBuild file /Users/williamlee/Documents/temp/console/obj/console.csproj.nuget.g.props.
+         Generating MSBuild file /Users/williamlee/Documents/temp/console/obj/console.csproj.nuget.g.targets.
+         Writing assets file to disk. Path: /Users/williamlee/Documents/temp/console/obj/project.assets.json
+         Restored /Users/williamlee/Documents/temp/console/console.csproj (in 131 ms).
```

Comment:
you might want to take that out, won't CredScan flag this?

Reply:
Removed